### ISSUE TITLE
FileSystemAlreadyExistsException in a multi modul project

### ISSUE THREAD

**Issue by dbuchhorn** [acanda/spring-banner-plugin#208]
Hello,

we use the banner maven plugin in a multi module project. If the projekt is build in parallel then sometimes a FileSystemAlreadyExistsException is thrown. The problem is located in method GenerateMojo.getBuildInFonts(). Because this method is called from different threads on the same time. The problen can be fixed by a synchronize block arround the code in this method.
````
        synchronized (RootPath.class) {
            try (RootPath rootPath = new RootPath()) {
                return rootPath.walkReadableFiles(FigFontResources.class, ".flf")
                        .map(path -> path.getFileName().toString())
                        .map(name -> name.substring(0, name.length() - 4))
                        .toList();
            } catch (IOException | URISyntaxException e) {
                throw new MojoFailureException("Cannot collect names of build-in fonts.", e);
            }
        }
````
Can you include a patch for the problem in the next release version?

Thank you

**Comment by acanda** [comment]
Hi Dirk, thanks for reporting this. I just released version 1.6.0 which hopefully fixes the problem. It should be available on Maven Central in an hour or two. Please let me know if this eliminates the FileSystemAlreadyExistsException in your project.

**Comment by dbuchhorn** [comment]
Hi Philip,

it works. Thank you for your fast response.
